cli/server/getAvailable.js

```diff
@@ -22,7 +22,8 @@ const getAvailableDatasets = async (localDataPath) => {
       !file.endsWith("_meta.json") &&
       !file.endsWith("_tip-frequencies.json") &&
       !file.endsWith("_root-sequence.json") &&
-      !file.endsWith("_seq.json")
+      !file.endsWith("_seq.json") &&
+      !file.endsWith("_measurements.json")
```

Comment:
There are a few other places in other repos which will probably want this same change. Should grep, but off the top of my head, at least the Nextstrain CLI and nextstrain.org repos.

Reply:
(Not saying you have to do this search, just noting it should get done.)